Explorer/Assets/DCL_RenderPass_DrawSkyBox.cs

```diff
@@ -0,0 +1,227 @@
+    {
+        //public static RTHandle k_CameraTarget
+        const string profilerTag = "Custom Pass: DrawSkyBox";
+        private Material m_Material_Draw;
+        private ProceduralSkyBoxSettings_Draw m_Settings_Draw;
+        RTHandle m_SkyBoxCubeMap_RTHandle;
+        RTHandle m_cameraColorTarget_RTHandle;
+        RTHandle m_cameraDepthTarget_RTHandle;
+        //private ScriptableRenderer m_Renderer = null;
+        Mesh m_cubeMesh = null;
+
+        private const string k_SkyBoxCubemapTextureName = "_SkyBox_Cubemap_Texture";
+        private static readonly int s_SkyBoxCubemapTextureID = Shader.PropertyToID(k_SkyBoxCubemapTextureName);
+
+        private enum ShaderPasses
+        {
+            Scenery = 0,
+            Sky = 1
+        }
+
+        internal DCL_RenderPass_DrawSkyBox()
+        {
+
+        }
+
+        internal void Setup(ProceduralSkyBoxSettings_Draw _featureSettings, Material _material, RTHandle _cameraColorTarget, RTHandle _cameraDepthTarget, RTHandle _SkyBox_Cubemap_Texture)
+        {
+            this.m_Material_Draw = _material;
+            this.m_Settings_Draw = _featureSettings;
+            this.m_cameraColorTarget_RTHandle = _cameraColorTarget;
+            this.m_cameraDepthTarget_RTHandle = _cameraDepthTarget;
+            this.m_SkyBoxCubeMap_RTHandle = _SkyBox_Cubemap_Texture;
+        }
+
+        // This method is called before executing the render pass.
+        // It can be used to configure render targets and their clear state. Also to create temporary render target textures.
+        // When empty this render pass will render to the active camera render target.
+        // You should never call CommandBuffer.SetRenderTarget. Instead call <c>ConfigureTarget</c> and <c>ConfigureClear</c>.
+        // The render pipeline will ensure target setup and clearing happens in a performant manner.
+        public override void OnCameraSetup(CommandBuffer cmd, ref RenderingData renderingData)
+        {
+
+        }
+
+        public override void Configure(CommandBuffer cmd, RenderTextureDescriptor cameraTextureDescriptor)
+        {
+            ConfigureClear(ClearFlag.None, Color.white);
+            MakeCube();
+        }
+
+        // Here you can implement the rendering logic.
+        // Use <c>ScriptableRenderContext</c> to issue drawing commands or execute command buffers
+        // https://docs.unity3d.com/ScriptReference/Rendering.ScriptableRenderContext.html
+        // You don't have to call ScriptableRenderContext.submit, the render pipeline will call it at specific points in the pipeline.
+        public override void Execute(ScriptableRenderContext context, ref RenderingData renderingData)
+        {
+            if (this.m_Material_Draw == null)
+            {
+                Debug.LogErrorFormat("{0}.Execute(): Missing material. DCL_RenderPass_DrawSkyBox pass will not execute. Check for missing reference in the renderer resources.", GetType().Name);
```

Comment:
Let's introduce a new category and use our own logger instead: it's described here: https://github.com/decentraland/unity-explorer/wiki/Diagnostics#logs-management